### DISCUSSION TITLE
Task lists: how do I fix errors

### DISCUSSION THREAD

**Discussion by Urbantat** [community/community#110177] [Projects and Issues]
### Select Topic Area

Bug

### Body

I am trying to create an issue with many task lists. Unfortunately, I am getting this error message in every task list. I have seen error messages when the character limit was reached within a task list or when the formatting was broken. However, this error is new to me and I would like to fix ist. Could you help me out? Thanks 🙏 
<img width="692" alt="Screenshot 2024-02-26 at 11 11 12" src="https://github.com/community/community/assets/97022182/dd906299-240d-41a0-81b0-18d69e9692e7">

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Urbantat ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. I've gone ahead and moved it for you. Good luck!

**Reply by Urbantat** [reply]
Thanks so much, @queenofcorgis ❤️
